app/views/dev_only/index.html.erb

```diff
@@ -0,0 +1,4 @@
+Assignment
```

Comment:
this should be in the dev_only component

Reply:
Correct. As with the controller this was my mistake and ended up in Mariko's code - it is already gone in the job-assignments branch.